src/content/en/tools/workbox/modules/workbox-webpack-plugin.md

```diff
@@ -2,31 +2,28 @@ project_path: /web/tools/workbox/_project.yaml
 book_path: /web/tools/workbox/_book.yaml
 description: The module guide for workbox-webpack-plugin.
 
-{# wf_updated_on: 2018-01-30 #}
+{# wf_updated_on: 2018-01-31 #}
 {# wf_published_on: 2017-12-15 #}
 {# wf_blink_components: Blink>ServiceWorker #}
 
 # Workbox webpack Plugins  {: .page-title }
 
 {% include "web/tools/workbox/_shared/alpha.html" %}
 
-Workbox provides to [webpack](https://webpack.js.org/), one that will
+Workbox provides two [webpack](https://webpack.js.org/) plugins: one that will
 generate a complete service worker for you and one that will generate a list
```

Comment:
I like to avoid unnecessary use of “will”. How about:

```
Workbox provides two [webpack](https://webpack.js.org/) plugins: one that
generates a complete service worker for you and one that generates a list
of assets to precache that is injected into a service worker.
```

Reply:
I'll updated it.